packages/console/src/utils/regex.ts

```diff
@@ -0,0 +1,3 @@
+// TODO - LOG-1876: Share Regex Between Logto Core and Front-End
+export const emailRegEx = /^\S+@\S+\.\S+$/;
```

Comment:
we can import emailRegEx from 'core'?

Reply:
No, we can't.
Add an issue to track this. LOG-1876